test/model/internal_link_test.dart

```diff
@@ -54,6 +54,37 @@ void main() {
     }
```

Comment:
3010732 `internal_link_test: Create 'parseInternalLink full url cases' test`

We usually use `internal_link test` for changes like this. `git log --oneline -- path/to/file` is a way to look for predecessors.